addons/web_editor/static/lib/odoo-editor/src/toolbar/Toolbar.js

```diff
@@ -0,0 +1,6 @@
+/** @odoo-module **/
+export class Toolbar{
+  constructor() {
```

Comment:
What's the plan here ? Wait after the merge to put all toolbar code here ?

Reply:
removed